cryptography/src/commonMain/kotlin/com/wire/kalium/cryptography/IDs.kt

```diff
@@ -72,15 +72,18 @@ data class CryptoQualifiedClientId(
 
 data class WireIdentity(
     val clientId: CryptoQualifiedClientId,
-    val handle: String,
+    val handle: String, // handle format is "{scheme}%40{handle}@{domain}", example: "wireapp://%40hans.wurst@elna.wire.link"
     val displayName: String,
     val domain: String,
     val certificate: String,
     val status: CryptoCertificateStatus,
     val thumbprint: String,
     val serialNumber: String,
     val endTimestamp: Long
-)
+) {
+    val handleWithoutSchemeAtSignAndDomain: String
+        get() = handle.substringAfter("://%40").removeSuffix("@$domain")
```

Comment:
i assume the %40 is  because it is url encoded,, so can we decode it first and then split?

Reply:
@MohamadJaara `URLDecoder` is from Android, do we have any multiplatform encoding solution?

Reply:
I will just merge it like this for now, we can update it later.